site/webpack.config.js

```diff
@@ -64,6 +64,12 @@ Encore
     .addEntry('planet', [
         './assets/js/planet.js'
     ])
+    .addEntry('notification', [
```

Comment:
Try to be consistent: the template is named "notification_list", how about using the same name here?